xml/System.Collections.Generic/List`1+Enumerator.xml

```diff
@@ -72,6 +72,8 @@
   
  <xref:System.Collections.Generic.List%601.Enumerator.Current%2A> returns the same object until <xref:System.Collections.Generic.List%601.Enumerator.MoveNext%2A> is called. <xref:System.Collections.Generic.List%601.Enumerator.MoveNext%2A> sets <xref:System.Collections.Generic.List%601.Enumerator.Current%2A> to the next element.  
   
+ The order of enumeration of list items proceeds from index zero to the last item of the list.
```

Comment:
Looks correct, even though it's probably not the best wording to describe this. Perhaps something along the lines of

> Lists are enumerated ordered by index, starting from the element with index zero up to the last item in the list.

I defer to @gewarren for choice of words.

Reply:
And move it before line 71.